### PR TITLE
Updating sponsor list

### PR DESCRIPTION
**Pull request by crazy4pi314**

### CODE REVIEW

**Review by nathanshammah — DISMISSED** [review]
Thanks @crazy4pi314 for adding the link to the blog post directly. If the sponsors are listed in tiers here, this will need yearly updates. Would wait for @willzeng approval before merging.

**Comment by crazy4pi314** [comment]
@willzeng, if you get a chance, could you let us know if this works for the supporter update? Thanks!

**Review by willzeng — APPROVED** [review]

**Pull request merged**
